CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - [#352](https://github.com/nf-core/metatdenovo/pull/352) - Updated some modules (@erikrikarddaniel).
 
+- [#368] (https://github.com/nf-core/metatdenovo/pull/368) - Added eukulele database name in filenames.
```

Comment:
I have put my items in reverse order. I.e. this would go at the top. I don't know what's the common pattern, nor if it matters at all. Add your GitHab id.